### ISSUE TITLE
[Bug]: Make page title update on showcase app

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shinyGovstyle#185]
### Overview

Currently we have a static page title, however we have multiple 'pages' within that app that have separate H1 headers, which then go out of sync with the page title. The lack of change in page title means that not all users are notified of the changes of content on the page.

### Steps to Reproduce

Go to the showcase app, and inspect the title element in the HTML.

### Expected Behavior

We should update the page title each time the h1 changes, e.g.

shinyGovstyle component showcase | tables and tabs
shinyGovstyle component showcase | text inputs

Maybe also worth thinking about condensing the initial title so it's more concise too - though need to make sure that's not at the cost of understandability.

### Additional Information

Can either pull in another dependency like https://github.com/ashbaldry/shinytitle/, or do our own title update function using JS (the latter might be better for long term maintainability, and it's pretty simple to add), e.g.

```
document.title="New Title";
document.getElementById("readTitle").innerHTML = document.title;
```
